### PR TITLE
Catalog of my things complete -  Ruby Capstone

### PR DESCRIPTION
**Pull request by coderacheal**
### **In this pull request**

- Created a main.rb file that will serve as your app entry-point.
- Present the user with a list of options to perform.
- Let users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.
- The following options should be available:
- (List all books, list all music albums, list all movies, list of games, list all genres (e.g 'Comedy', 'Thriller'), list all labels (e.g. 'Gift', 'New'), list all authors (e.g. 'Stephen King'), list all sources (e.g. 'From a friend', 'Online shop'), add a book, add a music album, add a movie, add a game)
- All data was preserved by saving collections in .json files.

Please find a link to the video presentation [here](https://drive.google.com/file/d/1TNaMKYGhFpPJW8yeP2DaY7UNCLG5AKDw/view?usp=sharing)

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

A number of sub projects found on the Kanban board are not approved! Kindly ensure to give reviews to each other's assigned task, ensure all tasks are approved and then request for another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

A number of sub projects found on the Kanban board are not approved! Kindly ensure to give reviews to each other's assigned task, ensure all tasks are approved and then request for another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by coderacheal** [comment]
@rloterh  We have already merged and resolved all merge conflicts so this cannot be done. If there is a way, please recommend it and we will have it done

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  last_played_at DATE
+)
+
+create table Authors (
+  id  INT generated always as identity,
+  first_name varchar(500),
+  last_name varchar(500),
+  primary key(id),
+
+  foreign key(id) references item(id)
```
Comment (AnselemOdims) [inline database/schema.sql:38]: - [x] Good job working on the schema queries but please kindly ensure you include indexes for all foreign keys.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @team, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by jerryowusu — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

weldone


## Highlights
App working fine :heavy_check_mark: 
Data stored succesfully :heavy_check_mark: 
All tests are passing :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**